src/views/TopologyDisplay.vue

```diff
@@ -0,0 +1,340 @@
+    this.open = [this.columnItems[0].id]
+    this.tabs = this.getTopologyTabs(nodes)
+
+  }
+
+
+  async updateLocations(locationIds: string[]) {
+    if (this.filterIds === undefined) return
+    if (locationIds.length === 0) {
+      if (this.topologyNode?.displayGroups !== undefined) {
+        await this.showDisplayGroupsTimeSeries()
+      } else {
+        this.showTimeSeries = false;
+      }
+      return
+    }
+    const [displays, requests] = await fetchTimeSeriesDisplaysAndRequests(this.webServiceProvider, this.filterIds, locationIds)
+    this.displays = displays
+    this.plots = []
+    await this.updateTimeSeries(requests, {startTime: this.startTime, endTime: this.endTime, thinning: true})
+    this.showTimeSeries = true
+  }
+
+  @Watch('selectedItem')
+  async onPlotChanged(): Promise<void> {
+    this.displays = this.allDisplays[this.selectedItem];
+    await this.loadTimeSeries(this.selectedItem);
+  }
+
+  @Watch('nodeId')
+  async onNodeChange(): Promise<void> {
+
+    const topologyNode: TopologyNode | undefined = this.topologyMap.get(this.nodeId)
+    if (topologyNode === undefined) return
+    this.showMap = topologyNode?.gridDisplaySelection?.plotId !== undefined || topologyNode.filterIds !== undefined
+    this.showTimeSeries = topologyNode?.displayGroups !== undefined
+
+    this.$nextTick(function () {
+      this.topologyNode = topologyNode //this is needed otherwise the property topologyNode is not passed to the explorer component when showMap was false
+      this.filterIds = topologyNode?.filterIds
+      this.wmsLayerId = topologyNode?.gridDisplaySelection?.plotId
+
+
+      if (!this.showTimeSeries) return
+      this.showDisplayGroupsTimeSeries();
+    })
+
+
+  }
+
+  private async showDisplayGroupsTimeSeries() {
+    const filter = {} as TopologyActionFilter;
+    filter.nodeId = this.nodeId;
+    const response: ActionsResponse = await this.webServiceProvider.getTopologyActions(filter);
+    this.selectedItem = 0;
+    this.allDisplays = [];
+    this.requests = [];
+    this.plots = [];
+    this.displays = [];
+    for (const result of response.results) {
+      if (result.config === undefined) continue;
+      const display: DisplayConfig[] = [];
+      const title = result.config.timeSeriesDisplay.title ?? ''
+      result.config.timeSeriesDisplay.subplots?.forEach((subPlot, index) => {
+        display.push({
+          id: `${title}-${index}`,
+          types: [DisplayType.TimeSeriesChart, DisplayType.TimeSeriesTable],
+          class: 'single',
+          title: title,
+          config: timeSeriesDisplayToChartConfig(subPlot, title)
+        })
+      })
+      this.allDisplays.push(display);
+      this.requests.push(result.requests);
+      this.plots.push(title)
+    }
+    this.displays = this.allDisplays.length > 0 ? this.allDisplays[0] : [];
+    await this.loadTimeSeries(0);
+  }
+
+  createTopologyMap(nodes: TopologyNode[] | undefined, topologyMap: Map<string, TopologyNode>) {
```

Comment:
Deze hebben we ook in `ExplorerComponent`? Moet dat dan ergens naar een typescript-module misschien?